lms/templates/courseware/course_about_sidebar_header.html

```diff
@@ -42,7 +43,7 @@
             subject=_("Take a course with {platform} online").format(platform=platform_name),
             body=_("I just enrolled in {number} {title} through {platform} {url}").format(
                 number=course.number,
-                title=course.display_name_with_default_escaped,
+                title=urllib.quote_plus((course.display_name_with_default).encode('UTF-8')),
```

Comment:
Same as above.  It is really `subject` and `body` that should be url encoded.  Maybe break this up into `email_subject`, `email_body`, and `email_link`, where you have something like:
```
email_link = u"mailto:?subject={subject}&body={body}".format(
            subject=urllib.quote_plus(email_subject)),
            body=urllib.quote_plus(email_body),
```
NOTE: I am proposing renaming what was `email_subject` to `email_link`, which means that `email_subject` would only be the subject in the newer code.

Reply:
done